src/verify.c

```diff
@@ -253,11 +253,14 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 		progress_set_next_step("download_packs");
 		print("\n");
 		get_all_files(official_manifest, subs);
+		progress_complete_step();
```

Comment:
I think you don't need this progress_complete_step(), the download_packs step should have closed automatically once we are done downloading packs, and then you added a packs_extraction step which should already been completed too by now